docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -209,6 +209,13 @@ duration can be changed using the `quarkus.test.native-image-wait-time` system p
 to 300 seconds, use: `./mvnw verify -Pnative -Dquarkus.test.native-image-wait-time=300`.
 ====
 
+By default, native tests runs using the `prod` profile.
```

Comment:
IMO, that's confusing. Especially taking into account the way we are writing the tests for the quickstarts where the native test simply extends the JVM one. So if you have something in the test profile, the JVM one will pass and the native one will fail (unless you explicitly enable the profile for it, of course).

Reply:
@aloubyansky I learned that the hard way (dropping my prod database, oupsy...)
More discussion is happening here: https://github.com/quarkusio/quarkus/issues/4371